### ISSUE TITLE
Bug: Element selection only works if tree view is open

### ISSUE THREAD

**Issue by MarkusSteinbrecher** [bldrs-ai/Share#318]
I created notes for this project: https://bldrs.ai/share/v/gh/Swiss-Property-AG/Seestrasse-Public/main/SEESTRASSE.ifc/86/109/136/28708/41077

On note Nr3, the slab on the ground floor is selected and included in the permalink.

- if the tree view is selected, the element is selected already on the first note
- if the tree view is hidden, the element is not selected at all

Does element selection require the tree view to be open?

**Comment by MarkusSteinbrecher** [comment]
also: if I select any element (e.g. the roof) while I have the notes open; then clear selection and skip to the next note, the roof is selected again.

**Comment by OlegMoshkovich** [comment]
Hello.

**Comment by OlegMoshkovich** [comment]
This is a general bug that has been there since the beginning, not related to Notes.

**Comment by pablo-mayrgundter** [comment]
I'm wondering if this is still happening after #328?  I can now click around on different comments' cameras and not reset what element is selected.  This is probably a problem itself, but not seeing any interaction with tree being open or not.

**Comment by MarkusSteinbrecher** [comment]
it seems to be working fine now, I can select elements without the tree

**Comment by pablo-mayrgundter** [comment]
Cool.  Ok, opening different issue for that.. it's simpler too.

**Comment by pablo-mayrgundter** [comment]
Hey @MarkusSteinbrecher could you review and close if fixed now.

**Comment by MarkusSteinbrecher** [comment]
it is still the same behavior as I commented above: "it seems to be working fine now, I can select elements without the tree. What is still there is this: Open notes view and select an element; deselect the element with the "clear selection" item; go to the next note and the element is selected again."

Selection works if tree view is closed. But if you select and deselect an element and skip through notes, the element is selected again. If this issue was just about the tree view, I guess it can be closed and we can create another issue.
